Invalid escape sequence in doc-strings.

Proposed fixes:
* Some doc-strings contain wrong text snippets: `True\False` &rarr; `True/False`  
  See #19 
* Fixed reference to `pyTooling`  
  See #20

This causes errors when using `itertree`:
```raw
  C:\Program Files\Python3.11\Lib\ast.py:50: in parse
    return compile(source, filename, mode, flags,
E     File "C:\Program Files\Python3.11\Lib\site-packages\itertree\itree_mathsets.py", line 580
E       """
E       ^^^
E   SyntaxError: invalid escape sequence '\F'
```

--- a/src/itertree/itree_mathsets.py
+++ b/src/itertree/itree_mathsets.py
@@ -582,7 +582,7 @@ a       :rtype: set
         result is a iterable over the single results
         :param value: to be checked iterable value (single item check)
         :param vars_dict: variable replacement dict
-        :return: iterable True\False
+        :return: iterable True/False
         """
         pass
 
@@ -1471,7 +1471,7 @@ class mSetInterval(_mSetBase):
         result is a iterable over the single results
         :param value: to be checked iterable value (single item check)
         :param vars_dict: variable replacement dict
-        :return: iterable True\False
+        :return: iterable True/False
         """
 
         limits = self._get_lower_upper(vars_dict)
@@ -1757,7 +1757,7 @@ class mSetRoster(_mSetBase):
         result is a iterable over the single results
         :param value: to be checked iterable value (single item check)
         :param vars_dict: variable replacement dict
-        :return: iterable True\False
+        :return: iterable True/False
         """
         _replacement_vars = set()
         if vars_dict is not None and self._vars:
@@ -2111,7 +2111,7 @@ class mSetCombine(_mSetBase):
         result is a iterable over the single results
         :param value: to be checked iterable value (single item check)
         :param vars_dict: variable replacement dict
-        :return: iterable True\False
+        :return: iterable True/False
         """
 
         return [self.__contains__(v, vars_dict) for v in value]

